dso-l2/src/main/java/com/tc/l2/state/ConsistencyManagerImpl.java

```diff
@@ -132,25 +135,26 @@ public boolean requestTransition(ServerMode mode, NodeID sourceNode, Transition
     boolean allow = false;
     
     // activate voting to lock the voting members and return the number of server votes
-    int serverVotes = activateVoting(mode, newMode);
+    int serverVotes = activateVoting(mode, newMode, topology);
+    int peerServers = topology.getServers().size() - 1;
 
```

Comment:
topology is locked while voting is occurring correct?  How about filtering the list of active peers before voting starts and using that for endVoting?  I'm concerned that it is not clear here that topology will not change during a vote.